projenrc/benchmark-test.ts

```diff
@@ -149,6 +152,10 @@ export class BenchmarkTest {
               'aws cloudwatch put-metric-data --metric-name TSC-average --namespace JsiiPerformance --value ${{ steps.output_summary.outputs.duration-tsc }}',
               'aws cloudwatch put-metric-data --metric-name JSII-average --namespace JsiiPerformance --value ${{ steps.output_summary.outputs.duration-jsii }}',
               'aws cloudwatch put-metric-data --metric-name JSII-slowdown --namespace JsiiPerformance --value ${{ steps.output_summary.outputs.jsii-slowdown }}',
+              'aws cloudwatch put-metric-data --metric-name JSII-average --namespace JsiiPerformance --value ${{ steps.output_summary.outputs.duration-jsii }} ' +
+                `--dimensions JsiiVersion=${jsiiVersion}`,
+              'aws cloudwatch put-metric-data --metric-name JSII-slowdown --namespace JsiiPerformance --value ${{ steps.output_summary.outputs.jsii-slowdown }} ' +
+                `--dimensions JsiiVersion=${jsiiVersion}`,
```

Comment:
This should not take multiple AWS CLI calls. You can put multiple metric values in one call. Please use that mechanism.